src/client/mod.rs

```diff
@@ -287,6 +292,58 @@ impl<'a> ClientBuilder<'a> {
 
         self
     }
+
+    /// Sets how long the http client should wait for the entire response from discord API.
+    /// By default, an http request will never timeout, it could create a deadlock sometimes.
+    ///
+    /// *Info*:
```

Comment:
We use the word *note* in **bold** in the documentation when we want to state additional information that the user should be aware of.
```suggestion
    /// **Note**:
```